tests/apollo/test_skvbc.py

```diff
@@ -21,7 +21,7 @@
 from util.bft import with_trio, with_bft_network, KEY_FILE_PREFIX
 
 
-def start_replica_cmd(builddir, replica_id):
+def start_replica_cmd(builddir, replica_id, config):
```

Comment:
Can you make the `config` parameter optional, so that we don't need to pass it every time?

Reply:
I wanted to avoid this too, but I didn't figure out a way to do it.

The problem is that at the place start_replica_cmd is called there is no knowledge if the replica is read-only or not. Initially I tried to add new method - start_ro_replica_cmd() but this way the modification is even more intrusive.

Reply:
Here's the way the `start_replica_cmd()` (and stop_) are implemented in Hermes:
https://gitlab.eng.vmware.com/blockchain/vmwathena_blockchain/blob/master/hermes/suites/apollo_bft_tests.py#L54

Reply:
PS: the purpose of the Apollo/Hermes integration was to run the same BFT tests we have, in the context of a containerized VMware Blockchain deployment. You can find more details about this integration here:
https://confluence.eng.vmware.com/pages/viewpage.action?spaceKey=BLOC&title=Apollo+testing+framework#Apollotestingframework-[Apollo&Hermes]TestingBFTaspartofproductCI(viaTEE)